nilearn/surface/surface.py

```diff
@@ -895,6 +897,65 @@ def load_surf_mesh(surf_mesh):
     return mesh
 
 
+def load_surface(surface):
+    """Loads a surface.
+
+    Parameters
+    ----------
+    surface : List of numpy.ndarray or Surface
+        Surface to be loaded. This can be passed as:
+
+        - a list of two files (valid formats are .gii
+        .gii.gz or Freesurfer specific files such as
+        .orig, .pial, .sphere, .white, .inflated) containing:
+            - a surface mesh geometry
+            - surface data
+        - three Numpy arrays organized in a list with coordinates,
```

Comment:
sorry for commenting a bit late but why do we accept this format (flat sequence of 3 elements)? I think having many options adds confusion rather than usability. I thought it would necessarily be ((coordinates, faces), data) ?